Userland/Utilities/js.cpp

```diff
@@ -71,6 +71,7 @@
 
 RefPtr<JS::VM> vm;
 Vector<String> repl_statements;
+JS::Handle<JS::Value> last_value = JS::make_handle(JS::js_undefined());
```

Comment:
Not very nice to have this be a global but it was much more complicated storing it in the global object or some other "correct" place

Reply:
It already has a ton of global state so that's fine :^)